commands/music/music.js

```diff
@@ -1,49 +1,40 @@
-const {prefix} = require("./../../config");
-const {join} = require("./../basic/join");
-const ytdl = require("ytdl-core");
-const {youtubeSearch} = require("./youtubeSearch");
+const {prefix} = require("./../../config")
+const {youtubeQueue} = require("./musicUtils/youtubeQueue") //goes thru queue and plays songs recursively //params = (queue, connection)
 
-
-
-const music = (client) => {    
+const music = (client, queue) => {
     client.on("message", (message) => {
         if (!message.guild){
             return
         }
         if(message.author.bot){
             return
         }
-        msg = message.content;
+        const msg = message.content
         
-        const searchterm = msg.slice(4, message.content.length);
-        const args = msg.split(" ");
-
+        const searchterm = msg.slice(6, message.content.length)
+        const args = msg.split(" ")
         
-        if(args[0] === prefix + "play" || args[0] === prefix + "p"){
-            if(message.member.voiceChannel){
-                
-                
+     
+        // let command = resolve_command()
+
+        // // look for js enum
+        // switch(command){
+        //     case const.Play:
+        //     break;
+        //     case const.Skip:
+        //     if (music_dispatcher){
+        //         md.end();
+        //     }
+
+
+        if(args[0].toLowerCase() === prefix + "play"){
```

Comment:
regex

Reply:
see previous
I do agree that regex is a good solution to this problem, but not only would I have to rewrite alot of code, but toLowerCase() seems to be a perfectly viable solution for the time being.